src/common-css/_common-mixins.scss

```diff
@@ -936,3 +936,597 @@ $sap-font-sizes: (
   }
   @content;
 }
+
+// COLORS
+$sap-colors: sapbrandcolor,
```

Comment:
Having so much "sap" is repetitive. We could remove "sap" from colors and in classes, it would be like this `sap-background-color-baseColor` instead of  `sap-background-color-sapBaseColor`